doc/mgba.6

```diff
@@ -0,0 +1,241 @@
+.Op Fl c Ar cheatfile
+.Op Fl l Ar loglevel
+.Op Fl p Ar patchfile
+.Op Fl s Ar n
+.Op Fl v Ar moviefile
+.Ar file
+.Sh DESCRIPTION
+.Nm
+is a Game Boy Advance emulator.
+The options are as follows:
+.Bl -tag -width Ds
+.It Fl 1
+Scale the window 1\(mu.
+.It Fl 2
+Scale the window 2\(mu.
+.It Fl 3
+Scale the window 3\(mu.
+.It Fl 4
+Scale the window 4\(mu.
+.It Fl 5
+Scale the window 5\(mu.
+.It Fl 6
+Scale the window 6\(mu.
+.It Fl b Ar biosfile , Fl -bios Ar biosfile
+Specify a BIOS file to use during boot.
+If this flag is omitted,
+.Nm
+will use the BIOS specified in the configuration file,
+or a high\(hylevel emulated BIOS if none is specified.
+.It Fl c Ar cheatfile , Fl -cheats Ar cheatfile
+Apply cheat codes from
+.Ar cheatfile .
+.It Fl d
+Start emulating via the command\(hyline debugger.
+.It Fl f
+Start the emulator full\(hyscreen.
+.It Fl g
+Start a
+.Xr gdb 1
+session.
+By default the session starts on port 2345.
+.It Fl l Ar loglevel
+Log messages during emulation to
+.Dv stdout .
+.Ar loglevel
+is a bitmask defining which types of messages to log:
+.Bl -inset
+.It 1
+fatal errors
+.It 2
+errors
+.It 4
+warnings
+.It 8
+informative messages
+.It 16
+debugging messages
+.It 32
+stub messages for unimplemented features
+.It 256
+in\(hygame errors
+.It 512
+software interrupts
+.It 1024
+emulator status messages
+.It 2048
+serial I/O messages
+.El
+The default is to log warnings, errors, fatal errors, and status messages.
+.It Fl p Ar patchfile , Fl -patch Ar patchfile
+Specify a patch file in BPS, IPS, or UPS format.
+.It Fl s Ar n , Fl -frameskip Ar n
+Skip every
+.Ar n
+frames.
+.It Fl v Ar moviefile , Fl -movie Ar moviefile
+Play back a movie of recording input from
+.Ar moviefile .
+.El
+.Sh CONTROLS
+The default controls are as follows:
+.Bl -hang -width "Frame advance" -compact
+.It A
+.Cm x
+.It B
+.Cm z
+.It L
+.Cm a
+.It R
+.Cm s
+.It Start
+.Aq Cm Enter
+.It Select
+.Aq Cm Backspace
+.It Load state
+.Cm F1 Ns \(en Ns Cm F9
+.It Save state
+.Ao Cm Shift Ac Ns \(hy Ns Cm F1 Ns \(en Ns Cm F9
+.It Frame advance
+.Ao Cm Ctrl Ac Ns \(hy Ns Cm n
+.El
+.Sh DEBUGGER
+When
+.Nm
+is run with the
+.Fl d
+option, the command\(hyline debugger is enabled.
+It supports the following commands:
+.Pp
+.Bl -tag -compact -width 1
+.It Cm b Ns Oo Cm reak Oc Ar address
+.It Cm b Ns Oo Cm reak Oc Ns Cm /a Ar address
+.It Cm b Ns Oo Cm reak Oc Ns Cm /t Ar address
+Set a breakpoint \(en ARM
+.Pq Ql /a ,
+Thumb
+.Pq Ql /t ,
+or the current CPU mode \(en at
+.Ar address .
+.It Cm c Ns Op Cm ontinue
+Continue execution.
+.It Cm d Ns Oo elete Oc Ar address
+Delete a breakpoint at
+.Ar address .
+.It Cm dis Ns Oo Cm asm Oc Ar address Op Ar count
+.It Cm dis Ns Oo Cm asm Oc Ns Cm /a Ar address Op Ar count
+.It Cm dis Ns Oo Cm asm Oc Ns Cm /t Ar address Op Ar count
+.It Cm dis Ns Oo Cm assemble Oc Ar address Op Ar count
+.It Cm dis Ns Oo Cm assemble Oc Ns Cm /a Ar address Op Ar count
+.It Cm dis Ns Oo Cm assemble Oc Ns Cm /t Ar address Op Ar count
```

Comment:
The `address` expression is optional--it will disassemble the current instruction if omitted.